bin/README.hermit.md

```diff
@@ -0,0 +1,7 @@
+# Hermit environment
```

Comment:
There is a way of commenting on a commit message but I forgot how.
I think it should be:
```
Gen-command: hermit init --no-git
```
(we need some verification tooling ;))